org_qubes_os_initial_setup/ks/qubes.py

```diff
@@ -99,7 +99,9 @@ class QubesData(AddonData):
 
     bool_options = (
         'system_vms', 'default_vms', 'whonix_vms', 'whonix_default', 'usbvm',
-        'usbvm_with_netvm', 'skip')
+        'usbvm_with_netvm', 'skip', 'fedora_available', 'debian_available',
+        'whonix_available'
```

Comment:
I don't think those three are part of the configuration - rather system state. On the other hand - selected lvm pool is.